crm_deduplicate_acl/models/res_partner.py

```diff
@@ -0,0 +1,16 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    @api.returns('self', lambda value: value.id)
+    def message_post(self, **kwargs):
```

Comment:
This changes upstream method signature. Use `*args, **kwargs`

Reply:
I copied from upstream, so are you sure?

Reply:
Yep, see: https://github.com/odoo/odoo/blob/622b1d23706aa5fdebeb0a6a467e0e993355efb6/addons/mail/models/mail_thread.py#L1770-L1772